modules/alloy/src/test/ts/browser/position/SelectionInFramePositionTest.ts

```diff
@@ -72,6 +76,33 @@ UnitTest.asynctest('SelectionInFramePositionTest', (success, failure) => {
       });
     };
 
+    const cAssertBelowSelection = NamedChain.bundle((data: any) => {
+      const frame = data.classic.element;
+      const root = SugarElement.fromDom(frame.dom.contentWindow.document.body);
+      const popup = data.popup;
+      const path = data.path3;
```

Comment:
This name needs to be passed in so this is reusable like the other functions here.

- [ ] Pass the path name in to the `cAssertBelowSelection` function.